### PR TITLE
🌱 Uplift cluster api to v1.1.5 and k8s version to v1.23.5 in fetch binaries

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by lentzi90 — APPROVED** [review]

File: hack/fetch_ext_bins.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ if [[ -n "${TRACE}" ]]; then
   set -x
 fi
 
-k8s_version=1.23.3
+k8s_version=1.23.5
```
Comment (furkatgofurov7) [inline hack/fetch_ext_bins.sh:29]: why not 1.23.8?

Comment (kashifest) [reply hack/fetch_ext_bins.sh:29]: Unavailable

**Review by furkatgofurov7 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**